Playground/Program.cs

```diff
@@ -16,9 +16,15 @@ public static void Main(string[] args)
             Console.ReadKey();
         }
 
+        private static string GetInput(string msg)
```

Comment:
Personally, I'd just load this via command line arguments. It'll achieve the same thing and be more flexible overall (in terms of debugging).
Not going to fail the review over this though.